src/customer-stories.html

```diff
@@ -0,0 +1,337 @@
+---
+layout: default
+permalink: /customer-stories
+title: Customer stories
+footer-class: absolute-footer
+---
+
+{% include clipped-header.html %}
+
+<div class="row justify-content-center justify-content-lg-start">
+  <div
+    class="clipped-header background-yellow-4 offset-lg-1 col-lg-10 col-11 mt-4 mt-md-5 mb-4 mb-lg-3 py-4 py-md-4 px-4 px-md-0"
+  >
+    <div class="offset-md-2 col-md-8 py-3 py-md-5 px-2 px-md-2">
+      <h1 class="mb-2">Our customers</h1>
+      <p class="mb-3 pb-3">
+        The California Integrated Travel Project (Cal-ITP) is working with transit agencies across the state (and country!) to
+        promote a modern and consistent transportation experience throughout California. Each and every transit agency in the
+        state can become a Cal-ITP customer. We provide free technical assistance and support to help agencies implement
+        Contactless Payments, Transit Benefits, GTFS and more.
+      </p>
+      <div class="row d-none d-lg-flex">
+        <div class="col-lg-8">
+          <span class="small-caps border-0 bg-transparent fw-bold p-0">Why it matters</span>
+        </div>
+        <div class="col-lg-4">
+          <span class="small-caps border-0 bg-transparent fw-bold p-0">Network of agencies</span>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-lg-4 d-flex justify-content-center flex-column">
+          <span class="small-caps border-0 bg-transparent fw-bold p-0 d-lg-none">Why it matters</span>
+          <span class="fs-0 d-block">1.88M</span>
+          <p>
+            Daily transit rides in CA<sup id="fnref:1" role="doc-noteref">
+              <a class="link-body-emphasis fw-normal text-decoration-none small" href="#fn:1" rel="footnote">1</a>
+            </sup>
+          </p>
+        </div>
+        <div class="col-lg-4 d-flex justify-content-center flex-column">
+          <span class="fs-0 d-block">300+</span>
+          <p>Transit agencies in CA; all can leverage Cal-ITP</p>
+        </div>
+        <div class="col-lg-4 d-flex justify-content-center flex-column">
+          <span class="small-caps border-0 bg-transparent fw-bold p-0 d-lg-none">Network of agencies</span>
+          <figure>
+            <img
+              src="/images/stories-map-agencies.png"
+              alt="image of map of California with XX agencies supported by Cal-ITP highlight across the state."
+              class="stories-map"
+            />
+          </figure>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<section class="row">
+  <div class="offset-lg-1 col-lg-10">
+    <div class="row flex-lg-row justify-content-center justify-content-lg-start mt-lg-5">
+      <div class="col-11 order-lg-2 offset-lg-1 col-lg-3">
+        <div class="sticky-top stories-toc" tabindex="1">
+          <div class="d-none d-lg-flex flex-column gap-3 mb-md-4 mb-3">
+            <span class="small-caps border-0 bg-transparent fw-bold p-0">Topics</span>
```

Comment:
I'm not sure what the best practice is for headings in a sidenav. I guess I feel like these could be `h2` but IDK.

* `Topics`
* `On this page`

Reply:
Taking a cue from https://www.w3.org/WAI/tutorials/menus/ and going to mark this up with a `<nav>` and aria-labeled by, and also changing the `ul` to an `ol` - becuase the order matters.